holiday-calendar-eur/src/main/java/com/github/davejoyce/calendar/impl/HolidayCalendarServiceCA.java

```diff
@@ -0,0 +1,152 @@
+                .rollable(false)
+                .build();
+        final Holiday thanksgiving = Holiday.builder()
+                .name("Thanksgiving Day")
+                .description("National day for giving thanks")
+                .type(Holiday.Type.FLOATING)
+                .rollable(false)
+                .observance(new Thanksgiving())
+                .build();
+        final Holiday remembranceDay = Holiday.builder()
+                .name("Remembrance Day")
+                .description("Commemoration of armed forces members who have died in the line of duty")
+                .type(Holiday.Type.FIXED)
+                .monthDay(Month.NOVEMBER, 11)
+                .rollable(true)
+                .build();
+        final Holiday christmas = Holiday.builder()
+                .name("Christmas Day")
+                .description("Christmas Day")
+                .type(Holiday.Type.FIXED)
+                .monthDay(Month.DECEMBER, 25)
+                .rollable(false)
+                .build();
+        final Holiday boxingDay = Holiday.builder()
+                .name("Boxing Day")
+                .description("Day after Christmas")
+                .type(Holiday.Type.FIXED)
+                .monthDay(Month.DECEMBER, 26)
+                .rollable(false)
+                .build();
+
+        return HolidayCalendar.builder()
+                .code(CODE)
+                .name(NAME)
+                .dateRoll(dateToRoll -> {
+                    final Optional<LocalDate> newYearsDayDate = newYearsDay.dateForYear(dateToRoll.getYear());
+                    final Optional<LocalDate> canadaDayDate = canadaDay.dateForYear(dateToRoll.getYear());
+                    final Optional<LocalDate> remembranceDayDate = remembranceDay.dateForYear(dateToRoll.getYear());
+                    final boolean isNewYearsDay = newYearsDayDate.isPresent() && dateToRoll.equals(newYearsDayDate.get());
+                    final boolean isCanadaDay = canadaDayDate.isPresent() && dateToRoll.equals(canadaDayDate.get());
+                    final boolean isRemembranceDay = remembranceDayDate.isPresent() && dateToRoll.equals(remembranceDayDate.get());
+                    if (dateToRoll.getDayOfWeek() == DayOfWeek.SUNDAY) {
+                        return (isNewYearsDay || isCanadaDay || isRemembranceDay) ? dateToRoll.plusDays(1L) : dateToRoll;
+                    }
+                    return dateToRoll;
+                })
+                .weekendDays(STANDARD_WEEKEND)
+                .holiday(newYearsDay)
+                .holiday(familyDay)
+                .holiday(goodFriday)
+                .holiday(easterMonday)
+                .holiday(victoriaDay)
+                .holiday(canadaDay)
+                .holiday(labourDay)
+                .holiday(nationalDayForTruthAndReconciliation)
+                .holiday(thanksgiving)
+                .holiday(remembranceDay)
+                .holiday(christmas)
+                .holiday(boxingDay)
```

Comment:
Maybe `holiday` should take a varargs parameter to avoid repeating it?

Reply:
I'll have to look into getting Lombok generated builder class to provide a `holidays(Holiday... holidays)` or a `holidays(Collection<Holiday> holidays)` method.